Fix "Variable 'exec_prefix' not defined in '.../lib/pkgconfig/libsasl2.pc'"
https://bugzilla.cyrusimap.org/show_bug.cgi?id=3831
--- libsasl2.pc.in.orig	2012-10-12 09:05:48.000000000 -0500
+++ libsasl2.pc.in	2013-12-19 04:27:47.000000000 -0600
@@ -1,4 +1,7 @@
-libdir = @libdir@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
 
 Name: Cyrus SASL
 Description: Cyrus SASL implementation
@@ -6,3 +9,4 @@
 Version: @VERSION@
 Libs: -L${libdir} -lsasl2
 Libs.private: @LIB_DOOR@ @SASL_DL_LIB@ @LIBS@
+Cflags: -I${includedir}
